app/controllers/users_controller.rb

```diff
@@ -8,22 +8,32 @@ def show
 
   # GET /users/1/edit
   def edit
+    @tab_select = params[:tab]
+    if @tab_select.present?
```

Comment:
@ChrisBr Initially even I did the same. I used bootstrap tab to render different partial for sake of simplicity. But the problem that I faced with Bootstrap tabs are as follows:

- It just add `.active` class to a current tab. i.e  Hides the partials of inactive tab
- When I tried submitting the form with this approach. Inactive tab fields acted as hidden fields to the form and also got submitted. 
This is the issue which @differentreality was talking about in previous PR #1180 
>Perhaps. The user, however, will be saving data that are not visible (ie data on the other tab will be saved as well).
 
To solve above issues, Option I had were as follows:
- Implement separate form in a separate tab to detach it from rest of the inactive tab fields. I really didn't like this approach
- Use Ajax to add and remove necessary and unnecessary fields respectively. 

Looking forward to suggestion.